gmm_client/src/client.erl

```diff
@@ -1,106 +1,118 @@
 %%%-------------------------------------------------------------------
-%%% @author Piotr Świderski
-%%% @copyright (C) 2021, <COMPANY>
 %%% @doc
-%%%
+%%%  Level of abstraction above raw http requests
```

Comment:
```suggestion
%%%  Level of abstraction over raw http requests
```

Ale i tak nie do końca mi się podoba taki opis modułu client.erl. Może lepiej by było pójść w stronę czegoś w stylu: "zbiór funkcji wykonujących zapytania do publicznego API wystawianego przez zonę" i nie mówić że to HTTP, bo i po co

Reply:
W sumie racja